### ISSUE TITLE
Would be nice if languages had a link to their repo/webpage

### ISSUE THREAD

**Issue by Zireael07** [ryusatgat/ryugod#8]
Currently all I can go on is google... which is bad for some of the more new/esoteric ones

**Comment by ryusatgat** [comment]
That's a good idea. Recently, I've been embedding information into "Hello, World" source code, but I'll consider it positively.

**Comment by ryusatgat** [comment]
The language selection dialog now includes the homepage and repository URLs(press F4 key or click language icon).
Currently, there is limited information available, but more details will gradually be added. For new languages, this information will be included.

**Comment by Zireael07** [comment]
Much appreciated (I see a lot of links now), but still some languages are missing links (at quick and pretty random glance, EmojiCode, Forte, Neko, NetRexx, NewtonScript, Oforth, Pocketpy, REXX, Wisnia, Wu, zkl)

**Comment by ryusatgat** [comment]
Yes, either the webpage "OR" the repository is registered (not both). The reason is that it's too much for me to handle alone. I'll fill it in little by little. Thanks to you, I've added Glance, though it's not a language. I have added information for all the mentioned languages.

**Comment by Zireael07** [comment]
Oh, "random glance" was an expression, not a language ;) Thanks
